packages/reports/src/Components/Form/CompositeCalendar/composite-calendar-mobile.tsx

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import { Button, DatePicker, Icon, InputField, MobileDialog, Text } from '@deriv/components';
 import { localize } from '@deriv/translations';
 import { toMoment } from '@deriv/shared';
+import { TInputDateRange } from '../../../Types';
```

Comment:
Please add `Types` alias in the `packages/reports/build/constants.js` file so you could get benefit of using absolute path
```suggestion
import { TInputDateRange } from 'Types';
```